CHANGELOG.rst

```diff
@@ -1,3 +1,7 @@
+0.2.2 (2016-03-30)
+==================
+* Bug fixes
```

Comment:
@percyperez I'm thinking it would be nice to be a bit more specific here. "Refresh token bugfixes" maybe.